Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:20.04
+LABEL do-not-remove=""
+
+RUN apt-get update \
+	&& apt-get -y install software-properties-common \
+		curl \
+		wget \
+		gnupg2 \
+		ca-certificates \
+		apt-transport-https \
+		gettext \
+	&& rm -r /var/lib/apt/lists/*
+
+RUN wget -qO - https://www.aptly.info/pubkey.txt | apt-key add - \
+	&& add-apt-repository "deb http://repo.aptly.info/ squeeze main"
+
+RUN apt-get update \
+    && apt-get install -y aptly python3-pip \
+    && python3 -m pip install awscli
+
+COPY .aptly.conf /home/github/.aptly.conf
```

Comment:
`/home/github` directory was never created :-|

is `github` an user? (otherwise `/home` doesn't make sense).

Reply:
copy to this destination makes not so much sense true because as i understand the docker action will create a path to github/home which will gets treated as their home directory `-v "/var/actions_runner/_work/_temp/_github_home":"/github/home"` this is also the reason why i need to put the .aplty.conf in the end to /github/home instead what we normaly think to /home/github

So in the end i can copy the conf and entrypoint to any location and will change it since this confuses